index.html

```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
+                <div class="project-button pro-btn-one">
                     <button>See Project</button>
                 </div>
+                
             </div>
         </section>
         <section class="about">
-            <div class="about-title">
+            <div class="about-title n-about-one">
                 <h3>About <br>Myself</h3>
             </div>
-            <div class="about-description">
+            <div class="about-description n-desc-one">
                 <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
                     of my
                     work and experience! If you like what you see and have a project you need coded, don’t hestiate to
                     contact
                     me.</p>
                 <p class="connect">Let's connect</p>
             </div>
-            <div class="about-social">
-                <ul>
+            <div class="about-social n-social-one">
+                <ul class="about-s-one">
                     <li><a href="#"><img src="./images/github.png"></a></li>
                     <li><a href="#"><img src="./images/linkedin.png"></a></li>
                     <li><a href="#"><img src="./images/hand.png"></a></li>
                     <li><a href="#"><img src="./images/twitter.png"></a></li>
                     <li><a href="#"><img src="./images/m.png"></a></li>
                 </ul>
             </div>
-            <div class="about-resume">
+            <div class="about-resume n-resume-one">
                 <button class="btn-resume">Get my resume</button>
             </div>
+            <div id="new-main-about">
+                <div class="about-title">
+                    <h3>About <br>Myself</h3>
+                </div>
+                <div class="about-description">
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                        of my
+                        work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                        contact
+                        me.</p>
+                    <p class="connect">Let's connect</p>
+                </div>
+                <div class="about-social">
+                    <ul>
+                        <li><a href="#"><img src="./images/github.png"></a></li>
+                        <li><a href="#"><img src="./images/linkedin.png"></a></li>
+                        <li><a href="#"><img src="./images/hand.png"></a></li>
+                        <li><a href="#"><img src="./images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./images/m.png"></a></li>
+                    </ul>
+                </div>
+                <div class="about-resume">
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+            </div>
             <div class="details">
```

Comment:
- [ ] Kindly add some space to the right of this section or adjust its width to match the template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215290977-28ba71c7-8a69-4ad0-a38f-928a1f68841c.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215290987-3afbffc6-6e8f-4b24-8db2-5715091acdb8.png)